obspy/io/nordic/core.py

```diff
@@ -714,7 +574,11 @@ def _read_picks(tagged_lines, new_event):
         if len(line) < 80:
             line = line.ljust(80)  # Pick-lines without a tag may be short.
         weight = line[14]
-        if weight == '_':
+        if weight not in ' 012349_':  # Long phase name
```

Comment:
This might be good to have as a CONSTANT at the top like the other header thingies (e.g. `ALLOWED_WEIGHTS`)

But ignore for now, we don't want another iteration of this PR for reviewing to speed up things.. :-)